datalad/plugin/wtf.py

```diff
@@ -219,7 +220,7 @@ def _describe_metadata_extractors():
 def _describe_dependencies():
     # query all it got
     [external_versions[k]
-     for k in tuple(external_versions.CUSTOM) + external_versions.INTERESTING]
+     for k in chain(external_versions.CUSTOM, external_versions.INTERESTING)]
```

Comment:
So this and the other `chain` call are needed because INTERESTING is now a list.  An alternative would have been `s/tuple/list/` in both spots.  Okay.

Reply:
yes. I decided to not rely on strict typing but rather on generic programming - that we just want to go through both regardless what they are

Reply:
To avoid any confusion: These sorts of comments are "thinking aloud" review comments.  I personally like seeing such comments from reviewers, both for my own patches and others.  They are not some sort of indirect way of me saying "you should have done X instead".